### ISSUE TITLE
Scheduler running jobs twice

### ISSUE THREAD

**Issue by candlerb** [zatosource/zato#436]
This is a zato-qs cluster (zato 2.0.3, ubuntu 14.04), reduced to one server and one gunicorn worker in that server.

There is a single scheduler job set to run every minute. `zato enmasse server1 --export-odb` shows the following:

~~~
...
 "scheduler": [
  {
   "cron_definition": null,
   "days": null,
   "extra": "",
   "hours": null,
   "id": 11,
   "is_active": true,
   "job_type": "interval_based",
   "minutes": 1,
   "name": "srv-cfflow.poll-all",
   "repeats": null,
   "seconds": null,
   "service": "srv-cfflow.poll-all",
   "service_id": 386,
   "service_name": "srv-cfflow.poll-all",
   "start_date": "2015-04-09T00:00:00",
   "weeks": null
  }
 ],
...
~~~

The service itself includes logging when it starts and finishes:

~~~
class PollAll(Service):
    def handle(self):
        self.logger.info("Starting PollAll")
        with closing(self.outgoing.sql.get(OUT_SQL).session()) as db:
            State.poll_all(db)
        self.logger.info("Poll Completed")   # log number of workflows/transitions?
        self.response.payload = "Poll Completed"
~~~

However, logs show zato is running it *twice* every minute.

~~~
$ tail -12 server1/logs/server.log
2015-04-23 14:42:00,692 - INFO - 2974:Dummy-776 - srv-cfflow.poll-all:22 - Starting PollAll
2015-04-23 14:42:00,698 - INFO - 2974:Dummy-776 - srv-cfflow.poll-all:22 - Poll Completed
2015-04-23 14:42:00,703 - INFO - 2974:Dummy-777 - srv-cfflow.poll-all:22 - Starting PollAll
2015-04-23 14:42:00,709 - INFO - 2974:Dummy-777 - srv-cfflow.poll-all:22 - Poll Completed
2015-04-23 14:43:00,694 - INFO - 2974:Dummy-800 - srv-cfflow.poll-all:22 - Starting PollAll
2015-04-23 14:43:00,700 - INFO - 2974:Dummy-800 - srv-cfflow.poll-all:22 - Poll Completed
2015-04-23 14:43:00,705 - INFO - 2974:Dummy-801 - srv-cfflow.poll-all:22 - Starting PollAll
2015-04-23 14:43:00,709 - INFO - 2974:Dummy-801 - srv-cfflow.poll-all:22 - Poll Completed
2015-04-23 14:44:00,694 - INFO - 2974:Dummy-820 - srv-cfflow.poll-all:22 - Starting PollAll
2015-04-23 14:44:00,701 - INFO - 2974:Dummy-821 - srv-cfflow.poll-all:22 - Starting PollAll
2015-04-23 14:44:00,717 - INFO - 2974:Dummy-820 - srv-cfflow.poll-all:22 - Poll Completed
2015-04-23 14:44:00,753 - INFO - 2974:Dummy-821 - srv-cfflow.poll-all:22 - Poll Completed
~~~

**Comment by dsuch** [comment]
Confirmed, the reason is that - erroneously - each server starts its own scheduler so a job is executed as many times as there are servers in a cluster. Working on a fix to be released in 2.0.4.

**Comment by dsuch** [comment]
In master since 174be654d595ceab7ace1f12e158113c3a36d357 - backported to support/2.0 and will be released in 2.0.4.
